script.js

```diff
@@ -247,3 +247,25 @@ sButton.forEach((button, index) => {
     section.scrollIntoView();
   });
 });
+
+// Client Side Validation
+
+const form = document.getElementById('contact_form');
+
+function clientValidationByEmail() {
+  const email = document.getElementById('email');
+
+  if (email.value !== email.value.toLowerCase()) {
+    alert('E-mail must be in lower case 😉');
+    return false;
+  }
```

Comment:
- [ ] From the lessons previously given to you, it's required that we follow [JavaScript Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and avoid using alert() and others you will find in that link. Kindly display an error text message close to the button when there is an uppercase letter in the email just as the requirement instructed and not alert(). 👍🏽

Reply:
Fixed.